serverless/installing_serverless/upgrading-serverless.adoc

```diff
@@ -0,0 +1,51 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="upgrading-serverless"]
+= Upgrading {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+:context: upgrading-serverless
+
+toc::[]
+
+If you have previously installed a Technology Preview version of {ServerlessProductName}, follow the instructions in this guide to upgrade to the latest version.
+
+In {ServerlessProductName} versions 1.5.0 and older, the only available Subscription Update Channel was `techpreview`.
+To upgrade to the latest version, you must update the channel to `preview-4.3`.
+
+[NOTE]
+====
+If you have selected Manual updates, you will need to complete additional steps after updating the channel as described in this guide. The Subscription’s upgrade status will remain *Upgrading* until you review and approve its Install Plan.
+For details, see the {product-title} documentation on xref:../../operators/olm-adding-operators-to-cluster.adoc#olm-adding-operators-to-a-cluster[adding Operators to a cluster].
+====
+
+.Prerequisites
+* You have installed a Technology Preview version of {ServerlessOperatorName}, and have selected Automatic updates during the installation process.
+* You have logged in to the {product-title} web console.
+
+.Procedure
+
+. Select the `openshift-operators` namespace in the {product-title} web console.
+. Navigate to the *Operators* → *Installed Operators* page.
+. Select the *{ServerlessOperatorName} Operator*.
+. Click *Subscription* → *Channel*.
+. In the *Change Subscription Update Channel* window, select `preview-4.3`, and then click *Save*.
+. Wait until all pods have been upgraded in the `knative-serving` namespace and the KnativeServing custom resource reports the latest Knative Serving version.
+
+.Verification steps
+
+To verify that the upgrade has been successful, you can check the status of pods in the `knative-serving` namespace, and the version of the KnativeServing CR.
+
+. Check the status of the pods by entering the following command:
++
+----
+oc get knativeserving.operator.knative.dev knative-serving -n knative-serving '-o=jsonpath={.status.conditions[?(@.type=="Ready")].status}
+----
++
+The previous command should return a status of `True`.
+
+. Check the version of the KnativeServing CR by entering the following command:
++
+----
+oc get knativeserving.operator.knative.dev knative-serving -n knative-serving '-o=jsonpath={.status.version}
```

Comment:
Should it be

`oc get knativeserving.operator.knative.dev knative-serving -n knative-serving -o=jsonpath='{.status.version}'`

with the `jsonpath` quoted?

Reply:
I'd imagine that's right, thanks!

Reply:
Let me check on it actually

Reply:
Yeah, as @jboxman said

Reply:
And the previous command should follow the same structure